pkg/opensearch/snapshot_tests/testdata/lucene_raw_data.query_input.json

```diff
@@ -0,0 +1,30 @@
+/* cSpell:disable */
```

Comment:
looks like this is causing the tests to fail.

you can add `"**/testdata/*.json"` to the `ignorePaths` in `cspell.config.json`

Reply:
thanks!!